packer/autocannon/build.pkr.hcl

```diff
@@ -0,0 +1,47 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.1.1"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "ubuntu" {
+  ami_name      = "autocannon"
+  instance_type = "t3.micro"
+  region        = "us-west-2"
+  source_ami_filter {
+    filters = {
+      name                = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"
+      root-device-type    = "ebs"
+      virtualization-type = "hvm"
+    }
+    most_recent = true
+    owners      = ["099720109477"]
+  }
+  ssh_username = "ubuntu"
+  tags = {
+    Name       = "Autocannon"
+    Team       = "NearForm"
+    Project    = "Elastic IPFS"
+    Repository = "https://github.com/elastic-ipfs/infrastructure"
+    ManagedBy  = "Packer"
+  }
+}
+
+build {
+  sources = [
+    "source.amazon-ebs.ubuntu"
+  ]
+
+  provisioner "shell" { # Install node and create base dir
+    inline = [
+      "curl -sL https://deb.nodesource.com/setup_${var.node_version}.x | sudo bash -",
+      "echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections",
+      "sudo apt-get update",
+      "sudo apt-get -y install nodejs git",
+      "git clone https://github.com/mcollina/autocannon"
```

Comment:
There is no need to clone this repo. What we need is a way of making the [actual test code](https://github.com/elastic-ipfs/bitswap-peer/tree/refactor/aws-client/test/e2e) to run inside this machine. Should we copy and run a script to start it (like I've done in bucket mirror) or create a workflow that delivers it somehow?

Reply:
Clone removed